neuralop/data/datasets/mesh_datamodule.py

```diff
@@ -3,7 +3,15 @@
 from typing import List, Union
 
 import numpy as np
-import open3d as o3d
+
+# import open3d for io if built. Otherwise,
+# the class will build, but no files will be loaded.
+try:
+    import open3d as o3d
+    open3d_available = True
+except ModuleNotFoundError:
+    open3d_available = False
```

Comment:
I don't think we need the additional flag: we're only wanting to import to extract the doc. If a user runs the code without the dependency, let's let it error rather than fail later.

Reply:
I think importing the module without the dependency fails